Address overhead concerns when sending 100% tx (#6790)
getsentry/sentry-docs: src/docs/product/performance/performance-at-scale/index.mdx

Retention priorities and your SDK sampling configuration act independently of each other, but work best when you send us as much data as you can. The more complete a picture of your application we have, the more accurately we can monitor your application health, detect problems faster, surface issues, and make better decisions about data retention on your behalf.

> To send 100% of your transaction events, set your [tracesSampleRate](/platform-redirect/?next=/performance/) in the SDK to 1.0. If you want to offset overhead, (with high-volume back-end applications, for example), set a lower value, or switch to using selective sampling and start filtering your transactions based on contextual data.

Author: ```suggestion
To send 100% of your transaction events, set your [tracesSampleRate](/platform-redirect/?next=/performance/) in the SDK to 1.0. If you want to offset overhead, (with high-volume back-end applications, for example), set a lower value, or switch to sampling selectively by using the <PlatformIdentifier name="traces-sampler" /> to filter your transactions based on contextual data.
```

> As your transaction volume goes up, the per-transaction pricing goes down accordingly, so you can truly scale affordably. Learn more in [Benefits of Performance at Scale](/product/performance/performance-at-scale/benefits-performance-at-scale/).

Reviewer: ```suggestion
As your transaction volume goes up, the per-transaction pricing will go down accordingly, making it possible to scale affordably. Learn more in [Benefits of Performance at Scale](/product/performance/performance-at-scale/benefits-performance-at-scale/).
```
